Esteban-Salas-Posada/tarea-1/style.css

```
@@ -0,0 +1,198 @@
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: right;
+    margin: 50px 20px 0 0;
+    font-weight: bold;
+}
+.magazine__parrafo--cuarto{
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: left;
+    margin: 0px 0 0 20px;
+}
+
+/* --------- Footer -------------- */
+
+.footer{
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 30px;
+}
+.footer__img{
+    max-width: 30%;
+    margin: 0 20px;
+}
+.footer__p{
+    width: 100%;
+    margin: 0px 20px 0 0;
+    font-size: 0.5rem;
+    text-align: center;
+    color: white;
+}
+
+/* 480px --------------------------- */
+@media (min-width: 468px){
+    .main{
+        background-color: #BBDDF6;
+        background-image: url(../img/modelo1.png);
+        background-position: center center;
+        background-repeat: no-repeat;
+        background-size: cover;
+        width: 80%;
+        height: 85vh;
+        margin: 0 auto;
+    }
+    .header{
+        max-width: 95%;
+        max-height: 25%;
+        margin: 0 auto;
+    }
+    .header__title{
+        font-size: 6rem;
+        color: #7FB0D4;
+        text-shadow: 3px 1px #333;
+        text-align: center;
+    }
+    .header__subtitle{
+        font-size: 3rem;
+        text-align: right;
+    }
+    .magazine__parrafo--primero{
+        font-size: 2.7rem;
+        text-shadow: 2px 1px #b1b1b1;
+        text-align: right;
+        margin: 50px 20px 0 0;
+    }
+    .magazine__parrafo--sgundo{
+        font-size: 2.7rem;
+        text-shadow: 2px 1px #b1b1b1;
+        text-align: left;
+        margin: 50px 0 0 20px;
+    }
+    .magazine__parrafo--tercero{
+        font-size: 2.7rem;
+        text-align: right;
+        margin: 50px 20px 0 0;
+        font-weight: bold;
+        text-shadow: 2px 1px #b1b1b1;
+    }
+    .magazine__parrafo--cuarto{
+        font-size: 2.7rem;
+        text-shadow: 2px 1px #b1b1b1;
+        text-align: left;
+        margin: 0px 0 0 20px;
+    }
+    .footer{
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+        margin-top: 30px;
+    }
+    .footer__img{
+        max-width: 30%;
+        margin: 0 20px;
+    }
+    .footer__p{
+        width: 100%;
+        margin: 0px 20px 0 0;
+        font-size: 2rem;
+        text-align: center;
+        color: white;
+    }
+}
+
+/* 1200px ------------------------------------------- */
+@media (min-width: 1200px){
+    .main{
+        background-color: #BBDDF6;
+        background-image: url(../img/modelo1.png);
+        background-position: center center;
+        background-repeat: no-repeat;
+        background-size: cover;
+        max-width: 1609px;
+        width: 60vw;
+        height: 95vh;
+        margin: 0 auto;
+    }
+}
+
+/* 1200px ------------------------------------------- */
+@media (min-width: 1200px){
+    .main{
+        background-color: #BBDDF6;
+        background-image: url(../img/modelo1.png);
```

@esteban-salas , this path is not working for me please fix
```suggestion
    background-image: url(img/modelo1.png);
```